### ISSUE TITLE
Issue_db - Users are able to view resumes of other users (P2)

### ISSUE THREAD

**Issue by joeljogy** [se2024-jpg/WolfTrack6.0#2]
Irrespective of the user logged in, they're able to view resumes of all users that are saved in the database. This is a serious security issue.

**Comment by girish1430** [comment]
Seems to be a priority issue. The query has to be filtered with the logged-in username, I believe

**Comment by joeljogy** [comment]
Agreed, will include user_id while querying on the database. Currently, this is not stored in the database while saving the resume. Hence, will need to make changes to first include user_id for every resume that is being saved to the database and then query based on the user_id. Definitely a high priority issue.
